platforms/extensibility/plugin-use/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequestApplicator.java

```diff
@@ -183,6 +187,7 @@ private static PluginResolution resolvePluginRequest(PluginResolver resolver, Pl
         PluginResolutionResult result;
         try {
             result = resolver.resolve(request);
+            LOGGER.info("Resolved plugin {}", request.getDisplayName());
```

Comment:
❓ Is adding this info log necessary? I feel that this would create a lot of noise

Reply:
I'd argue that it is becoming necessary with this PR adds additional ways of injecting plugins into a build.  I don't think it's typically too much extra info, as it's just once per plugin resolution, not plugin application.